proposals/csharp-12.0/collection-expressions.md

```diff
@@ -250,6 +259,8 @@ If the target type is an *array*, a *span*, a type with a *[create method](#crea
 * For each element in order:
   * If the element is an *expression element*, the initialization instance *indexer* is invoked to add the evaluated expression at the current index.
   * If the element is a *spread element* then one of the following is used:
+    * If the spread element expression is a *collection expression*, then the collection expression elements are evaluated in order as if the elements were in the containing collection expression.
+    * If the spread element expression is a *conditional expression*, then the condition is evaluated and if `true` the second-, otherwise the third-operand is evaluated as if the operand was an element in the containing collection expression.
```

Comment:
What if we did not specify treating conditional expressions differently during construction than other expressions, and leave compiler optimizations discretionary?